src/common/queries/scheduledOrders.ts

```diff
@@ -1,10 +1,10 @@
-import { Account } from '../../account'
-import { Address } from 'viem'
+import { Address, PublicClient } from 'viem'
 import { GRAPHQL_API_URL } from '../constants'
+import { Account } from '../../account'
 
 const query = `
-    query ($smartAccount: String) {
-      scheduledOrdersExecutionAddedQueries (where:{ smartAccount: $smartAccount}) {
+    query ($smartAccount: String, $chainId: Int) {
```

Comment:
nice is the indexer part for this built already and waiting for deployed address or do we still need to build it?

Reply:
@kopy-kat it is already built